app/javascript/components/rooms/room/room_settings/RoomSettings.jsx

```diff
@@ -10,15 +10,15 @@ import AccessCodes from './AccessCodes';
 import Modal from '../../../shared_components/modals/Modal';
 import DeleteRoomForm from '../forms/DeleteRoomForm';
 import useRoomConfigs from '../../../../hooks/queries/admin/room_configuration/useRoomConfigs';
+import useRolePermissions from '../../../../hooks/queries/admin/role_permissions/useRolePermissions';
 
 export default function RoomSettings() {
   const { friendlyId } = useParams();
   const roomSetting = useRoomSettings(friendlyId);
   const roomsConfigs = useRoomConfigs();
-
+  const rolePermissions = useRolePermissions();
```

Comment:
Instead of loading this, you can just do `currentUser.permissions.CanRecord == 'true'`